requirements.txt

```diff
@@ -1,4 +1,5 @@
-requests==2.18.4
-six==1.11.0
-tenacity==4.10.0
-oauth2client==4.1.2
+requests>=2.19.1
+six>=1.11.0
+oauth2client>=4.1.2
+tenacity>=4.10.0
+setuptools_scm==3.*
```

Comment:
Should this be `'setuptools_scm>=3.1.0,<4'` to be consistent with setup.py?

Reply:
Will change. (I thought >=, < is not working in requirements.txt, I was wrong :()